demos/calibration/calibration-pubmed-node-classification.ipynb

```diff
@@ -1674,307 +303,13 @@
     "We now do the same for the node attributes we want to use to predict the subject. These are the feature vectors that the Keras model will use as input. The dataset contains attributes 'w-*' that correspond to words found in that publication."
```

Comment:
Since loading the node features now happens in the load() method, you have correctly removed the code that performs the operation described here. The dangling comment can be removed as well.

Reply:
Good point, although I think this comment was already outdated/dangling, since I don't think it was "doing the same" as any of the code working with the target variable.